postgresql_blueprints/db_utils.py

```diff
@@ -0,0 +1,84 @@
+from sqlalchemy import create_engine, text
+
+
+def setup_connection(args):
+    """
+    Set up a connection to the database using the keyword arguments provided.
+    This will override system defaults.
+
+    :param args: keyword arguments provided by the user
+    :return: a connection to the database
+    """
+    db_string = create_connection_string(args)
+    if connect_args := format_connect_args(args):
```

Comment:
The walrus operator (`:=`) is not supported in 3.7, but >3.8. If the Postgres blueprints are 3.7 then you will need to change the Python version and also the check to see if the dependencies are good for 3.9 as well